vllm/model_executor/models/jamba.py

```diff
@@ -211,7 +214,9 @@ def __init__(self,
 
         num_experts = config.layers_num_experts[layer_idx]
         ffn_layer_class = JambaMoE if num_experts > 1 else JambaMLP
-        self.feed_forward = ffn_layer_class(config, quant_config=quant_config)
+        self.feed_forward = ffn_layer_class(config,
+                                            quant_config=quant_config,
+                                            prefix=f"{prefix}.ffn")
```

Comment:
I think this needs to be feed_forward per https://huggingface.co/ai21labs/AI21-Jamba-1.5-Mini?show_file_info=model.safetensors.index.json
```suggestion
                                            prefix=f"{prefix}.feed_forward")
```